File: src/main/java/model/games/Traditional.java
package model.games;

import model.LeapFrog;
import model.boards.TraditionalBoard;

public class Traditional extends LeapFrog {
    public Traditional() {
        super("Traditional");
    }

    public void initialize(int rows, int cols) {
        super.initialize(rows, cols);
        this.board = new TraditionalBoard().createBoard(rows, cols);
    }
}

File: docs/scripts/games/Traditional.js
import TraditionalBoard from "../boards/TraditionalBoard.js";
import NoMoveHigherScore from "../endOfGame/NoMoveHigherScore.js";
import LeapFrog from "../LeapFrog.js";
import OrthogonalMoves from "../possibleMoves/OrthogonalMoves.js";
import IncreaseByCapture from "../updateScore/IncreaseByCapture.js";

export default class Traditional extends LeapFrog {
    constructor() {
        super("Traditional");
    }
    initialize(rows, cols) {
        super.initialize(rows, cols);
        this.board = new TraditionalBoard().createBoard(rows, cols);
        this.possibleMoves = new OrthogonalMoves(this);
        this.updateScore = new IncreaseByCapture();
        this.endOfGame = new NoMoveHigherScore(this);
    }
}